src/extension.ts

```diff
@@ -51,7 +51,7 @@ export function activate(context: vscode.ExtensionContext) {
           Authorization: `Basic ${auth}`,
         },
       }
-      const schemaURL = jenkinsURL + "configuration-as-code/schema"
+      const schemaURL = jenkinsURL + addTrailingSlash("configuration-as-code/schema")
```

Comment:
it is not `configuration-as-code/schema` that needs the trailing slash.
It's jenkins URL and should only add it if does not end with a trailing slash.

Reply:
In the Readme, I added that the user enter the Jenkins url in the format that it contains a trailing slash.Fine to adding it here though.

Reply:
not every user will do it, same reason why Jenkins Core has the same fix :)